docs/vars.list

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<vars>
+    <var name="dokkaVersion"
+         value="1.7.20"
+         type="string"/>
+    <var name="kotlinVersion"
+         value="1.7.20"
+         type="string"/>
+</vars>
```

Comment:
Question to the website / tech writers team: I see that coroutines have no `vars.list`, but they use vars throughout documentation.

As I understand it, there's a common `vars.list` that is used throughout all of kotlinlang? Do we need to remove this one from our project, or can it stay?